### PR TITLE
feat(frontend): Added support for replacetext to mongo5 translator

### PR DESCRIPTION
**Pull request by lukapeschke**
Signed-off-by: Luka Peschke <luka.peschke@toucantoco.com>

### CODE REVIEW

File: src/lib/translators/mongo5.ts [hunk 1]
```diff
@@ -38,6 +40,23 @@ function transformDateExtract(step: Readonly<S.DateExtractStep>): object {
   return transformDateExtractFactory(ADVANCED_DATE_EXTRACT_MAP_MONGO_5)(step);
 }
 
+/** transforms a 'replacetext' step into corresponding mongo steps. $replaceAll was only added in  */
+/** mongo 4.4 */
```
Comment (Sanix-Darker) [inline src/lib/translators/mongo5.ts:44]: 👍🏽

**Review by Sanix-Darker — APPROVED** [review]
LGTM !

File: src/lib/translators/mongo5.ts [hunk 2]
```diff
@@ -38,6 +40,23 @@ function transformDateExtract(step: Readonly<S.DateExtractStep>): object {
...
+function transformReplaceText(step: Readonly<S.ReplaceTextStep>): MongoStep {
+  return {
+    $set: {
+      [step.searchColumn]: {
```
[the remark this answers is not in the document]
Comment (lukapeschke) [reply src/lib/translators/mongo5.ts:48]: I think you're getting too used to python :smile: In JS what's before the `:` is expected to be a property name, and having a `.` in the middle is invalid syntax :smile:

[the remark this answers is not in the document]
Comment (Sanix-Darker) [reply src/lib/translators/mongo5.ts:48]: 🙈 shame on meeee !

**Pull request merged**